### PR TITLE
New: Add support for tooltip API (#88)

### LINKED ISSUE #88: Tooltip label missing
### Subject of the issue
The close extension is missing the tooltip label feature now available in core.

### Your environment
* FW 5.31.23
* LanguagePicker 5.3.0

### PR DESCRIPTION
**Pull request by joe-allen-89**
[//]: # (Please title your PR according to eslint commit conventions)
[//]: # (See https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-eslint#eslint-convention for details)

[//]: # (Link the PR to the original issue)
#88 

[//]: # (Delete Fix, Update, New and/or Breaking sections as appropriate)
### New
* Add support for tooltip API (fixes #88)

### CODE REVIEW

**Review by eleanor-heath — APPROVED** [review]

File: example.json [hunk 1]
```diff
@@ -48,5 +48,10 @@
 
     // to be added to the _extensions object in course.json
     "_languagePicker": {
-        "navigationBarLabel" : "Select course language"
+        "navigationBarLabel" : "Select course language",
+        "languageSelector": "Language selector",
+        "_navTooltip": {
+            "_isEnabled": true,
+            "text": "Language Picker"
```
Comment (oliverfoster) [inline example.json:55]: Is it a language picker or a language selector?

Comment (joe-allen-89) [reply example.json:55]: @oliverfoster languageSelector is in the templates and was in schemas but not example, happy to address in this PR though.

Comment (oliverfoster) [reply example.json:55]: I don't think you should change the schema property names. I meant that the text is both `"Language Picker"` and `"Language selector"`. You've added `"Language Picker"` but the original text says `"Language selector"`.

Comment (joe-allen-89) [reply example.json:55]: @oliverfoster do we want to leave it as Language Selector and update tooltip to match?

**Comment by danielghost** [comment]
There was already a branch to include this and the navigation API:
https://github.com/adaptlearning/adapt-contrib-languagePicker/issues/83

**Comment by oliverfoster** [comment]
It was in the wrong column, not ready for review @danielghost

**Comment by danielghost** [comment]
> It was in the wrong column, not ready for review @danielghost

The readme hadn't yet been updated to reflect the attribute changes.

File: example.json (hunk 1 above)
Comment (oliverfoster) [reply example.json:55]: Yea, probably easier.

**Comment by oliverfoster** [comment]
Dan's branch has the navigation button api, Joe's has the readme updates, both have the tooltip api. Can I suggest a merger?

**Comment by oliverfoster** [comment]
Pr for the merger.

I think we just need to revert `navBarLabel` to `navigationBarLabel`, to save unnecessarily changing the schema in a breaking way.

**Review by kirsty-hames — APPROVED** [review]
Works as expected thanks. Testing on Mac Safari, Chrome and FF.

**Review by oliverfoster — APPROVED** [review]

**Pull request merged**